br/backup-and-restore-overview.md

```diff
@@ -0,0 +1,123 @@
+
+| Feature | Issue | Solution |
+|  ----  | ----  | ----- |
+| Clustered index | [#565](https://github.com/pingcap/br/issues/565)       | Make sure that the value of the `tidb_enable_clustered_index` global variable during restoration is consistent with that during backup. Otherwise, data inconsistency might occur, such as `default not found` and inconsistent data index. |
+| New collation  | [#352](https://github.com/pingcap/br/issues/352)       | Make sure that the value of the `new_collations_enabled_on_first_bootstrap` variable during restoration is consistent with that during backup. Otherwise, inconsistent data index might occur and checksum might fail to pass. |
+| Global temporary tables | | Make sure to use BR v5.3.0 or later for backup and restoration. Otherwise, wrong table definition will be generated for global temporary tables. |
+
+However, even after you have ensured that the preceding features are consistently enabled or disabled during backup and restoration, compatibility issues might still occur due to the inconsistent internal versions or inconsistent interfaces between BR and TiKV/TiDB/PD. To avoid such cases, BR provides a built-in version check.
+
+#### Version check
+
+Before performing backup and restoration, BR compares and checks the TiDB cluster version and the BR version. If there is a major-version mismatch (for example, BR v4.x and TiDB v5.x), BR prompts a reminder to exit. To forcibly skip the version check, you can set `--check-requirements=false`.
+
+Note that skipping the version check might introduce incompatibility. The version compatibility mapping between BR and TiDB versions are as follows:
+
+| Backup version (vertical) \ Restoration version (horizontal) | Use BR nightly to restore TiDB nightly | Use BR v5.0 to restore TiDB v5.0 | Use BR v4.0 to restore TiDB v4.0 |
+|  ----  |  ----  | ---- | ---- |
+| Use BR nightly to back up TiDB nightly | ✅ | ✅ | ❌ (If a table with the primary key of the non-integer clustered index type is restored to a TiDB v4.0 cluster, BR will cause data error without warning.) |
+| Use BR v5.0 to back up TiDB v5.0 | ✅ | ✅ | ❌ (If a table with the primary key of the non-integer clustered index type is restored to a TiDB v4.0 cluster, BR will cause data error without warning.) |
+| Use BR v4.0 to back up TiDB v4.0 | ✅ | ✅  | ✅ (If TiKV >= v4.0.0-rc.1, and if BR contains the [#233](https://github.com/pingcap/br/pull/233) bug fix and TiKV does not contain the [#7241](https://github.com/tikv/tikv/pull/7241) bug fix, BR will cause the TiKV node to restart.)|
+| Use BR nightly or v5.0 to back up TiDB v4.0 | ❌ (If the TiDB version is earlier than v4.0.9, the [#609](https://github.com/pingcap/br/issues/609) issue might occur.)| ❌ (If the TiDB version is earlier than v4.0.9, the [#609](https://github.com/pingcap/br/issues/609) issue might occur.)| ❌ (If the TiDB version is earlier than v4.0.9, the [#609](https://github.com/pingcap/br/issues/609) issue might occur.) |
+
```

Comment:
This whole part is missing from the Chinese version. Is that intended?
![image](https://user-images.githubusercontent.com/41534398/173780207-a45883e3-c98b-4ee3-8f30-7f549253e611.png)

Reply:
According to confirmation with Ian, this information is covered in this table item: https://github.com/pingcap/docs/pull/8628/files#diff-5f5519a7f44e95526e3253e724fdc87ff3e57461ceae2bfd6317dc96775fe3fcR79.
![image](https://user-images.githubusercontent.com/89768198/175848912-22d10477-473e-414c-847f-d6f95066c05c.png)

Reply:
Fixed in https://github.com/pingcap/docs/pull/8628/commits/0d1ff8980803e77e4c15c6b78321add372489338#diff-5d65e67605cdab9570939131909b470ad54050d43dc181aacc0a832850ccbacaR230.